libdnf/dnf-package.cpp

```diff
@@ -89,6 +90,29 @@ dnf_package_get_priv(DnfPackage *pkg)
     return priv;
 }
 
+/**
+ * dnf_package_is_local:
+ * @pkg: a #DnfPackage *instance.
+ *
+ * Returns: %TRUE if the pkg is a pkg on local or media filesystem
+ *
+ * Since: 0.37.1
```

Comment:
Can you please bump the versions here, since `0.37.2` was already released...

Reply:
Will do.